### DISCUSSION TITLE
TPOT port 64297 is not working

### DISCUSSION THREAD

**Discussion by zzfabiozz** [telekom-security/tpotce#1464] [Q&A]
Hello guys do you have any idea how to fix this?

I'm trying to connect to the UI  but it's not working. This is the second time that I have installed the TPOT but the same problem. 

![image](https://github.com/telekom-security/tpotce/assets/43100584/eedcbf46-3e6c-4809-aaa7-d59ef0977530)

**Comment by Pelican9091** [comment]
You need to use https
